app/domain/operations/families/create_member.rb

```diff
@@ -55,6 +55,7 @@ def persist_person(person)
         vlp_document = person.consumer_role.vlp_documents.first
         consumer_role.active_vlp_document_id = vlp_document.id if vlp_document.present?
         person.save!
+        person.create_demographics_group
```

Comment:
adding here will update person record again. Lets move this to the operation: **app/domain/operations/people/transform_applicant_to_member.rb** and use the pattern present in this operation.